src/sentry/utils/redis.py

```diff
@@ -134,7 +138,7 @@ def cluster_factory():
                     # reset() after startup
                     #
                     # https://github.com/Grokzen/redis-py-cluster/blob/73f27edf7ceb4a408b3008ef7d82dac570ab9c6a/rediscluster/nodemanager.py#L385
-                    startup_nodes=deepcopy(hosts),
+                    startup_nodes=deepcopy(hosts_list),  # type: ignore[call-overload]
```

Comment:
Basically, `Any`, `dict[Any, Any]` is allowed. Having tighter types fails...

Reply:
this seems to pass mypy 🤔 
```python
import copy
from typing import Any

def f() -> None:
    x: list[dict[Any, Any]]


    copy.deepcopy(x)
```

Reply:
ah this is probably because our stubs file for rediscluster is incomplete -- if you add this parameter to `fixtures/stubs-for-mypy/rediscluster/__init__.pyi` it should be happier?

Reply:
Phew, nice catch.